### DISCUSSION TITLE
[Options Search]: activation and filter clearing by hot key, searching in section name and matches highlighting

### DISCUSSION THREAD

**Discussion by PetbkA** [doublecmd/doublecmd#1210] [Suggestions]
> Double Commander
> Version: **1.1.0 beta**
> Revision: 32
> Commit: **cd3a69c65**
> Build date: 2023/08/05
> Lazarus: 2.2.7.0
> FPC: 3.2.2
> Platform: **x86_64-Win64-win32/win64**
> OS version: **Windows 10 Home 22H2 x86_64**

Nice to have search in options, but there're some points that could be improved:
1. Can't activate search by hot key (tried _Ctrl + F_ and _F3_ + searched for command but found none). Would be nice to have command in _Hot Keys_;
2. Can't clear filter on hot key - _Esc_ quits options. Would be nice to have command in _Hot Keys_;
3. Search works only _inside_ section, but not for section name (tried with e.g. "wfx" and got empty result). Would be nice to search in section names too;
4. Matches aren't highlighted. Would be nice to [highlight them](https://github.com/doublecmd/doublecmd/assets/22843816/b38b1bcd-1f6b-4c84-a57f-0d7e3f8ecf61)

**Comment by alexx2000** [comment]
4. Impossible
